--- src/cloudpilot/Makefile.orig	2024-03-31 16:54:44
+++ src/cloudpilot/Makefile	2024-03-31 17:47:00
@@ -14,15 +14,15 @@
 
 WEBIDL_BINDER ?= $(PYTHON) $${EMSDK}/upstream/emscripten/tools/webidl_binder.py
 
 CFLAGS_NATIVE ?= \
 	-DENABLE_DEBUGGER -O2  -g $(shell sdl2-config --cflags)
 
 CXXFLAGS_NATIVE ?= $(CFLAGS_NATIVE)
 
 INCLUDE_EXTRA ?=
 
 LDFLAGS_NATIVE ?=  \
-	$(shell sdl2-config --libs) -lSDL2_image -lreadline -lboost_coroutine -ldl -lpthread
+	$(shell sdl2-config --libs) -lSDL2_image -lreadline -lboost_coroutine-mt -ldl -lpthread
 
 CFLAGS_COMMON := -Werror -Wextra -Wall -Wno-unused-parameter -Wno-pragma-pack -Wno-multichar -Wno-unknown-pragmas \
 	-Wno-missing-field-initializers -DEMULATION_LEVEL=EMULATION_UNIX
